.github/actions/slither-analysis/action.yml

```diff
@@ -5,44 +5,98 @@ inputs:
     description: Path to slither config file
     required: false
     default: ./slither.config.json
+  target:
+    description: Target directory or file for slither analysis
+    required: false
+    default: .
 runs:
   using: "composite"
   steps:
     - name: Slither Analyze
       id: slither
-      uses: crytic/slither-action@v0.4.1
-      if: github.event_name != 'pull_request'
-      continue-on-error: true
-      with:
-        slither-config: ${{ inputs.CONFIG }}
-        fail-on: pedantic
-        sarif: results.sarif
+      shell: bash
+      run: |
+        pip install slither-analyzer
+        echo "Running slither analysis..."
+        echo "Target: ${{ inputs.target }}"
+        echo "Config: ${{ inputs.CONFIG }}"
```

Comment:
This is purely a convention, but I like to use uppercase for org/repo/environment vars and secrets, and all lower-case for inputs or `needs/steps.X.outputs` vars, to easily identify their locality.